protocol/contracts/templegold/SpiceAuction.sol

```diff
@@ -206,25 +248,49 @@ contract SpiceAuction is ISpiceAuction, AuctionBase {
      * @notice Claim (retro) rewards for an epoch . Cannot claim for a live epoch auction
      * @param epochId Epoch to claim for
      */
-    function claim(uint256 epochId) external virtual override {
+    function claim(uint256 epochId) external payable virtual override {
```

Comment:
Oh - it didn't click that this would also need to receive ETH to do the LZ part.

How will that work from a UX & dapp perspective?

Reply:
That's why I was saying there's nothing in it for the claimer to pay eth for redemption update.
from a UX perspective, dapp would display info on why claimer pays eth for `claim`. Then dapp will check:
```
bidToken == templeGold && claimShouldNotifyTotalRedeemed  && !redeemedEpochs[epochId]
``` 
it's not fair for the first claimer imo unless we give them some incentive

Reply:
As agreed, i think you rugged this